compiler/src/lnntoamm/Event.ts

```diff
@@ -0,0 +1,66 @@
+import { LPNode } from "../lp";
+import Output from "./Amm";
+import Fn from './Fn';
+import Scope from "./Scope";
+import Type, { Builtin } from "./Types";
+import { genName, TODO } from "./util";
+
+var allEvents: Event[] = [];
```

Comment:
Nit: why not `let`?

Reply:
I can change it to `let`, it just felt wrong to me to use `let` with my go/rust background